initializr-service/src/main/resources/application.yml

```diff
@@ -962,6 +962,12 @@ initializr:
           versionRange: 1.3.0.RELEASE
           groupId: org.springframework.cloud
           artifactId: spring-cloud-stream-binder-kafka
+        - name: Kafka Streams
+          id: cloud-stream-binder-kafka-streams
+          description: Stream processing microservices with Kafka Streams API
+          versionRange: 1.3.0.RELEASE
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-stream-binder-kstream
```

Comment:
I would like two links added to this new dependency:

1. The link to a section in a ref guide that explains how it can be used
2. A link to a sample that demonstrates the getting started experience (basic usage).